egs/ami/asr1/run.sh

```diff
@@ -178,6 +178,12 @@ if [ ${stage} -le 1 ]; then
         data/${train_set}/feats.scp data/${train_set}/cmvn.ark exp/dump_feats/train ${feat_tr_dir}
     dump.sh --cmd "$train_cmd" --nj 10 --do_delta $do_delta \
         data/${train_dev}/feats.scp data/${train_set}/cmvn.ark exp/dump_feats/dev ${feat_dt_dir}
+    for rtask in ${recog_set}; do
+        feat_recog_dir=${dumpdir}/${rtask}/delta${do_delta}; mkdir -p ${feat_recog_dir}
+        dump.sh --cmd "$train_cmd" --nj 32 --do_delta $do_delta \
```

Comment:
`--nj 32` is too many. `--nj 10` should be fine.

Reply:
Just to confirm. Should I apply the same configuration to all the recipes? or just this one.

Reply:
same number of jobs to the dev (above). Please apply it to all recipes.